swamp/parsers/mtzparser.py

```diff
@@ -4,22 +4,43 @@
 from swamp.parsers.parser import Parser
 
 
-class MTZLabels(Enum):
+class MtzColumnLabels(Enum):
     """An enumerator that contains the regular expression used to detect the column labels of a given MTZ file"""
-
-    free = re.compile(r"[Ff][Rr][Ee][Ee]")
-    i = re.compile(r"[Ii]$")
-    sigi = re.compile(r"[Ss][Ii][Gg][Ii]$")
-    f = re.compile(r"[Ff][Pp]?$")
-    sigf = re.compile(r"[Ss][Ii][Gg][Ff][Pp]?$")
-    i_plus = re.compile(r"[Ii]\(\+\)")
-    sigi_plus = re.compile(r"[Ss][Ii][Gg][Ii]\(\+\)")
-    f_plus = re.compile(r"[Ff][Pp]?\(\+\)")
-    sigf_plus = re.compile(r"[Ss][Ii][Gg][Ff][Pp]?\(\+\)")
-    i_minus = re.compile(r"[Ii]\(-\)")
-    sigi_minus = re.compile(r"[Ss][Ii][Gg][Ii]\(-\)")
-    f_minus = re.compile(r"[Ff][Pp]?\(-\)")
-    sigf_minus = re.compile(r"[Ss][Ii][Gg][Ff][Pp]?\(-\)")
+    free = re.compile(r"^.*?[Ff][Rr][Ee][Ee].*([Ff][Ll][Aa][Gg])?")
```

Comment:
Change this from `free = re.compile(r"^.*?[Ff][Rr][Ee][Ee].*([Ff][Ll][Aa][Gg])?")` to `free = re.compile(r"^.*?[Ff][Rr][Ee][Ee].*")` to ensure SA_flag columns are not accidentally assigned to the free flag